distributions/otelcol-ebpf-profiler/manifest.yaml

```diff
@@ -0,0 +1,37 @@
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/ebpf-profiler
+  name: otelcol-ebpf-profiler
+  description: OpenTelemetry Collector for eBPF Profiling
+  version: 0.123.1
+  output_path: ./_build
+
+exporters:
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.123.0
+  - gomod: go.opentelemetry.io/collector/exporter/nopexporter v0.123.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.123.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.123.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.123.0
+
+processors:
+  - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.123.0
+  - gomod: go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.123.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor v0.123.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.123.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor v0.123.0
+
+receivers:
+  - gomod: go.opentelemetry.io/ebpf-profiler v0.0.0-20250408174414-ead430a15d49
+    import: go.opentelemetry.io/ebpf-profiler/collector
+    name: profilesreceiver
+
+providers:
+  - gomod: go.opentelemetry.io/collector/confmap/provider/envprovider v1.29.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/fileprovider v1.29.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/httpprovider v1.29.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/httpsprovider v1.29.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/yamlprovider v1.29.0
+
+# When adding a replace, add a comment before it to document why it's needed and when it can be removed
+replaces:
+  # see https://github.com/openshift/api/pull/1515
+  - github.com/openshift/api => github.com/openshift/api v0.0.0-20230726162818-81f778f3b3ec
```

Comment:
The other distributions don't need this, what about this distribution does?

Reply:
So doe the module itself.
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/go.mod#L192

Every time I've used this module so far, I needed to include this replace. I've been a bit surprised by that, but I assumed this was a known issue.